python/doc/math_notations.sty

```diff
@@ -6,12 +6,61 @@
 
 % The physical model
 \newcommand{\model}{g}
+\newcommand{\inputDim}{d}
+\newcommand{\outputDim}{d_Y}
+% domaine
+\newcommand{\inputDomain}{\mathcal{D}}
+% The surrogate model
+\newcommand{\metaModel}{\widetilde{\model}}
+
+% Probabilistic modeling
+\newcommand{\inputRV}{\vect{X}}
+\newcommand{\outputRV}{\vect{Y}}
+\newcommand{\inputMeasure}{\mu_{\inputRV}}
+\newcommand{\outputMeasure}{\mu_{\outputRV}}
+% Realization of the random vetor
+\newcommand{\inputReal}{\vect{x}}
+\newcommand{\outputReal}{\vect{y}}
+\newcommand{\RVZ}{\vect{Z}}
+\newcommand{\RVU}{\vect{U}}
+\newcommand{\dimZ}{{d_Z}}
+\newcommand{\dimU}{{d_U}}
+\newcommand{\realZ}{\vect{z}}
+\newcommand{\realU}{\vect{u}}
```

Comment:
Using the letter Z in the name _and_ in the content is pointless.
```suggestion
\newcommand{\standardRV}{\vect{Z}}  # The standard random vector
\newcommand{\uniformRV}{\vect{U}}
\newcommand{\standardDim}{{d_Z}}  # The dimension of the standard random vector
\newcommand{\dimU}{{d_U}}
\newcommand{\standardReal}{\vect{z}}  % An observation of the standard random vector
\newcommand{\realU}{\vect{u}}
```

Reply:
I don't agrre because the random vector Z is not always the one expressed in the standard space and the U random vector does not always follow a Uniform distribution.....

Reply:
Ok for the alternate $\LaTeX$ code. It seems to me that these changes were forgotten:

![image](https://github.com/openturns/openturns/assets/31351465/234a4dac-8fce-4ee9-9acb-5239ff060c29)